MFVNeutralino/test/TriggerStudies/filtercheck.py

```diff
@@ -13,10 +14,18 @@
 
 from JMTucker.MFVNeutralino.EventFilter import setup_event_filter
 sef = lambda *a,**kwa: setup_event_filter(process, *a, input_is_miniaod=True, **kwa)
+sef('pTrigger', mode = 'trigger jets only')
+sef('pTriggerMET', mode = 'trigger met only', name_ex = 'met')
 #sef('pTriggerBjets', mode = 'trigger bjets only',name_ex = 'bjets')
 #sef('pTriggerDispDijet', mode = 'trigger displaced dijet only',name_ex = 'displaced_dijet')
-sef('pTriggerOR', mode = 'trigger bjets OR displaced dijet', name_ex = 'bjets_OR_displaced_dijet')
+#sef('pTriggerOR', mode = 'trigger HT OR bjets OR displaced dijet', name_ex = 'HT_OR_bjets_OR_displaced_dijet')
+#sef('pJets',    mode = 'jets only novtx',   name_ex = 'NoVtx') # be sure to generate a different name for each subsequent use
+#sef('pNoJESUp', mode = 'jets only novtx',   name_ex = 'NoJESUp', event_filter_jes_mult = 0)
+#sef('pTriggerOR', mode = 'trigger bjets OR displaced dijet', name_ex = 'bjets_OR_displaced_dijet')
 #sef('pFull',    mode = 'jets only',         name_ex = 'Full') # uncomment to get efficiency of ntuple-level vertex filter
+sef('pTriggerMuons', mode = 'trigger muons only',name_ex = 'muons')
```

Comment:
I'm not sure what the study being done here was, but I am surprised that we turned on the MET trigger and disabled the bjets one here?